server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -140,6 +146,8 @@ export function configureWebSocketServices(
 		const scopeMap = new Map<string, string[]>();
 		// Map from client Ids to connection time.
 		const connectionTimeMap = new Map<string, number>();
+		// Map from client Ids to supportedFeatures
+		const supportedFeaturesMap = new Map<string, Record<string, any>>();
```

Comment:
`any`? It seems unlikely that we want `any`. At the very least `unknown`, but we can't do better than that? What could possibly be in those values?